src/backends/backend/python/python_interpreter.cc

```diff
@@ -744,9 +749,22 @@ TRITONBACKEND_ModelInstanceExecute(
   ni::ExecuteResponse execute_response;
 
   // Perform inference on the Python side
-  const auto err = instance_state->stub->Execute(
+  const auto status = instance_state->stub->Execute(
       &context, execute_request, &execute_response);
 
+  if (!status.ok()) {
+    return TRITONSERVER_ErrorNew(
```

Comment:
Lifecycle. In general once you have created and returned any response you can't return an error from this function. You must handle all response sending yourself.